### DISCUSSION TITLE
Onion skin by color.

### DISCUSSION THREAD

**Discussion by wandrinsheep** [keiwando/toonsquid#502] [Ideas]
Hi there so I’m currently working on a piece of animation that involves many pixel layers per drawing. I have to need to track each of these pixel layers’ motion but it all becomes a handful pretty quickly. So in scenarios where there are a lot of elements to be onion skinned allow only specified color(s) to be isolated in onion skin mode

**Comment by andydjames** [comment]
At the moment layers in the timeline can be enabled/disabled to show onion skins.
Would it be possible to add the ability to enable/disable onions skins at drawing layer level?
It would also be really cool to add additional icons from the drawing layer level alongside the visible/hidden [eye] icon e.g. lock, onion skinable...
